streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -122,14 +105,14 @@ public void commit() {
      */
     @Override
     public void suspend() {
-        log.info("{} Suspending", logPrefix);
+        log.debug("{} Suspending", logPrefix);
```

Comment:
@mjsax Just realized we already have an info log entry in onPartitionXX callbacks, so this should be fine. NVM.